maintainers/ALL-OWNERS

```diff
@@ -3,25 +3,19 @@
 
 approvers:
 - andfasano
-- ashughorla
 - bfournie
 - derekhiggins
 - dtantsur
 - elfosardo
 - furkatgofurov7
-- hroyrh
-- imain
+- honza
 - iurygregory
 - kashifest
-- knowncitizen
-- ogelbukh
-- ravipwaghmare
-- russellb
-- stbenjam
```

Comment:
as discussed offline, these and others removed were not moved to emeritus because they are not in the emeritus approvers list in the repos where they are maintainers so the list of people here are automatically fetched/adjusted by the script.